lib/components/molecules/ArticleThumbnail/ArticleThumbnail.style.js

```diff
@@ -0,0 +1,5 @@
+import { css } from 'styled-components';
+
+export default css`
```

Comment:
Can you define some basic styles for a styled version too? This is the final nail :)